bin/plugin/commands/build-plugins.js

```diff
@@ -0,0 +1,183 @@
+				const pluginSlug = plugins[ moduleDir ]?.slug;
+
+				if ( ! pluginVersion || ! pluginSlug ) {
+					log(
+						formats.error(
+							'The given module configuration is invalid, the module JSON object is missing the "version" and/or "slug" properties, or they are misspelled.'
+						)
+					);
+					return;
+				}
+
+				try {
+					// Copy module files from the folder.
+					const modulePath = path.join( '.', 'modules/' + moduleDir );
+					const buildModulePath = path.join( '.', 'build/' + pluginSlug );
+					try {
+						// Clean up build module files directory.
+						fs.rmSync( buildModulePath, { force: true, recursive: true } );
+
+						fs.copySync( modulePath, buildModulePath, { overwrite: true } );
+					} catch ( copyError ) {
+						log(
+							formats.error(
+								`Error copying files for plugin "${ pluginSlug }": "${ copyError }"`
+							)
+						);
+						continue;
+					}
+
+					// Update file content.
+					updatePluginHeader( {
+						modulePath: moduleDir,
+						slug: pluginSlug,
+						version: pluginVersion,
+						pluginPath: buildModulePath,
+					} );
+
+					// Update text domain.
+					updateModuleDetails( {
+						pluginPath: buildModulePath,
+						regex: '[\']performance-lab[\']',
+						result: `'${ pluginSlug }'`,
+					} );
+
+					// Update `@package`.
+					updateModuleDetails( {
+						pluginPath: buildModulePath,
+						regex: '@package\\s{1,}performance-lab',
+						result: '@package ' + pluginSlug,
+					} );
+				} catch ( error ) {
+					log(
+						formats.error(
+							`${ error }`
+						)
+					);
+				}
+			}
+		} catch ( jsonError ) {
+			log(
+				formats.error(
+					`Error parsing JSON string: "${ jsonError }"`
+				)
+			);
+		}
+	} );
+};
+
+/**
+ * Updates the `load.php` file header information.
+ *
+ * @param {Object} settings Plugin settings.
+ */
+async function updatePluginHeader( settings ) {
+	const { modulePath, version, slug, pluginPath } = settings;
+	// Specific module `load.php` file content.
+	const buildLoadFile = path.join( pluginPath, 'load.php' );
+	let buildLoadFileContent = '';
+	try {
+		buildLoadFileContent = fs.readFileSync( buildLoadFile, 'utf-8' );
+	} catch ( err ) {
+		log(
+			formats.error(
+				`Error reading the file "${ buildLoadFile }": "${ err }"`
+			)
+		);
+	}
+
+	const moduleHeader = await getModuleHeader( buildLoadFileContent );
+
+	// Get module header data.
+	const { name, description } = await getModuleDataFromHeader( moduleHeader );
+
+	const pluginHeader = `/**\n * Plugin Name: ${ name }\n * Plugin URI: https://github.com/WordPress/performance/tree/trunk/modules/${ modulePath }\n * Description: ${ description }\n * Requires at least: 6.1\n * Requires PHP: 5.6\n * Version: ${ version }\n * Author: WordPress Performance Team\n * Author URI: https://make.wordpress.org/performance/\n * License: GPLv2 or later\n * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html\n * Text Domain: ${ slug }\n *\n * @package ${ slug }\n `;
```

Comment:
Maybe you can break this into multiple lines for better readability? JS template strings can be broken apart into multiple lines I think. This way you can also get rid of all the `\n` usage.